File: backend/internal/model/database/profit.go
package database

type Profit struct {
	ID                  int64   `json:"id,omitempty"`
	PitchID             int64   `json:"pitch_id"`
	DeclaredBy          string  `json:"declared_by"`
	PeriodStart         string  `json:"period_start"`
	PeriodEnd           string  `json:"period_end"`
	TotalProfit         float64 `json:"total_profit"`
	DistributableAmount float64 `json:"distributable_amount"`
	Transferred         bool    `json:"transferred"`
	CreatedAt           string  `json:"created_at,omitempty"`
}

File: frontend/lib/types/profit.ts
export type Profit = {
	pitch_id: number, 
  total_profit: number, 
	period_start: string, 
	period_end: string
};

export type ProfitFromApi = {
	id: number;
	pitch_id: number;
	declared_by: string;
	period_start: string;
	period_end: string;
	total_profit: number;
	distributable_amount: number;
	transferred: boolean;
};
